test/hotspot/gtest/aarch64/asmtest.out.h

```diff
@@ -712,6 +712,7 @@
     __ stxp(r4, zr, zr, r5);                           //       stxp    w4, xzr, xzr, [x5]
     __ stxpw(r6, zr, zr, sp);                          //       stxp    w6, wzr, wzr, [sp]
     __ dup(v0, __ T16B, zr);                           //       dup     v0.16b, wzr
+    __ dups(v0, __ T2S, v1);                           //       mov     s0, v1.s[0]
```

Comment:
Please use the `DUP` AArch64 instruction here, rather than `MOV`.

Reply:
OK.